SignalServiceKit/src/Messages/DeviceSyncing/OWSOutgoingSentMessageTranscript.m

```diff
@@ -48,15 +49,21 @@ - (OWSSignalServiceProtosSyncMessageBuilder *)syncMessageBuilder
 
     OWSSignalServiceProtosSyncMessageSentBuilder *sentBuilder = [OWSSignalServiceProtosSyncMessageSentBuilder new];
     [sentBuilder setTimestamp:self.message.timestamp];
-    [sentBuilder setDestination:self.message.recipientIdentifier];
-    [sentBuilder setMessage:[self.message buildDataMessage:self.message.recipientIdentifier]];
+    [sentBuilder setDestination:self.recipientIdentifierForMessage];
+    [sentBuilder setMessage:[self.message buildDataMessage:self.recipientIdentifierForMessage]];
     [sentBuilder setExpirationStartTimestamp:self.message.timestamp];
 
     [syncMessageBuilder setSentBuilder:sentBuilder];
 
     return syncMessageBuilder;
 }
 
+// Note that this will return nil for group messages.
```

Comment:
This isn't a change in behavior, I've just pulled this method from `TSOutgoingMessage` to here, the only place we use it.

I'm going to make a card to investigate this.  It might be correct behavior, but it's strange that.